.semaphore/semaphore.yml

```diff
@@ -0,0 +1,22 @@
+version: v1.0
+name: Java
+agent:
+  machine:
+    type: e1-standard-2
+    os_image: ubuntu1804
+blocks:
+  - name: Gradle
+    task:
+      jobs:
+        - name: Build
+          commands:
+            - sem-version java 1.8

```

Comment:
You're now testing only Java 8, in Travis CI it was tested for 8, 9, 10 and 11.

Reply:
I'll add them again. but travis is currently so unreliable, really wanted to switch :)